examples/fuzzed_data_provider_fuzz_test.cc

```diff
@@ -14,11 +14,14 @@
 
 // A fuzz target that demonstrates the use of FuzzeddataProvider.
 
-#include <fuzzer/FuzzedDataProvider.h>
-
 #include <cstddef>
 #include <cstdint>
 
+// Work around a CI failure due to symbols from <limits> missing in the
+// FuzzedDataProvider.h header.
```

Comment:
Maybe mention here that we should remove this header here once we update to a Clang version more recent than https://github.com/llvm/llvm-project/commit/0cccccf0d2cbd707503263785f9a0407d3e2bd5e?

Reply:
Thanks for finding the root cause, done. Do you understand why this only became an issue now?

Reply:
> Do you understand why this only became an issue now?

That's a really good question. Unfortunately I can't think of a concrete good reason - the builds are supposed to be hermetic and deterministic. I suspect either changes in the Bazel version or the underlying Ubuntu image might have caused a change in compilation order, which in turn exposed this missing include.